### ISSUE TITLE
Implement navigating menus with keyboard

### ISSUE THREAD

**Issue by blairlyons** [simularium/simularium-website#347]
## Use Case
The menus cannot be accessed today by keys only. The menu interaction needs to be updated to support better accessibility.

## Acceptance Criteria
User navigating website with keyboard should be able to use return key or spacebar to prompt a menu open from buttons

## Details

Design Link: TBD (https://github.com/simularium/simularium-website/issues/296)
